package.json

```diff
@@ -24,19 +24,19 @@
     "@babel/preset-env": "^7.14.2",
     "@codexteam/ajax": "^4.0.1",
     "babel-loader": "^8.2.2",
-    "css-loader": "^5.2.4",
-    "eslint": "^6.1.0",
-    "eslint-config-codex": "github:codex-team/eslint-config",
-    "eslint-loader": "^2.2.1",
+    "css-loader": "^1.0.0",
+    "eslint": "^7.32.0",
+    "eslint-config-codex": "^1.6.4",
+    "eslint-webpack-plugin": "^3.1.1",
```

Comment:
Does it work?

Looking on the [official doc](https://webpack.js.org/plugins/eslint-webpack-plugin/):

> This is eslint-webpack-plugin 3.0 which works only with webpack 5. For the webpack 4, see the 2.x branch.

Reply:
yep. works good

need to get the latest version cause

![image](https://user-images.githubusercontent.com/15259299/143092587-21021cc1-1cfe-47f7-af28-0864bd3c9fb3.png)